src/pages/settings/Report/WriteCapabilityPage.js

```diff
@@ -38,24 +40,26 @@ function WriteCapabilityPage(props) {
 
     return (
         <ScreenWrapper includeSafeAreaPaddingBottom={false}>
-            <HeaderWithBackButton
-                title={props.translate('writeCapabilityPage.label')}
-                shouldShowBackButton
-                onBackButtonPress={() => Navigation.goBack(ROUTES.getReportSettingsRoute(props.report.reportID))}
-            />
-            <OptionsList
-                sections={[{data: writeCapabilityOptions}]}
-                onSelectRow={(option) => Report.updateWriteCapabilityAndNavigate(props.report, option.value)}
-                hideSectionHeaders
-                optionHoveredStyle={{
-                    ...styles.hoveredComponentBG,
-                    ...styles.mhn5,
-                    ...styles.ph5,
-                }}
-                shouldHaveOptionSeparator
-                shouldDisableRowInnerPadding
-                contentContainerStyles={[styles.ph5]}
-            />
+            <FullPageNotFoundView shouldShow={ReportUtils.isAdminRoom(props.report)}>
```

Comment:
Hi ✋ Coming from https://github.com/Expensify/App/issues/22013

We also want to restrict this for non admin policy.